deployment/pypi/setup.py

```diff
@@ -15,6 +17,34 @@
 else:
     raise NotImplementedError('current platform {} not supported'.format(os_type))
 
+class AutoCompletion(install):
+    def run(self):
+        COMP_URL = 'https://raw.githubusercontent.com/microsoft/nni/master/tools/bash-completion'
```

Comment:
it is not good to use master branch here

Reply:
why not pack it into python package?

Reply:
Sure, will change to a specific release branch.
If we pack it into python package, it is hard to locate the root of the python package, so I choose to directly download it.